dna/course/zomes/courses/code/src/lib.rs

```diff
@@ -72,7 +72,7 @@ mod courses {
             None => return Ok(None),
         }
     }
-
+    // sections_address - do I need to replicate this in the sections CRUD? Seems weird to update the course this way. 
```

Comment:
hi josh! Yep, we would need to have an update_section function as well.
update_course updates the entire course entry. This also makes it possible for update_course to technically either add or remove section in the sections field of course struct. However, if we do want to update the details of the section itself (which for now is only title), then we would need the update_section function :D